lib/src/subscriber.rs

```diff
@@ -39,6 +39,17 @@ impl<'a> std::future::Future for SubscriptionComplete<'a> {
     }
 }
 
+/// Stream of messages from [Publishers](super::publisher::Publisher)
+///
+/// The new function starts the process of subscribing to a publisher. A message could arrive at
+/// any point after the call completes. [Subscription::wait_for_subscription_complete] can be used to listen
+/// for the subscription acknowledgement.
+///
+/// All inbound messages are handled during the stream poll. This can mean that a subscriber not
+/// handling messages can be deemed in active and it's subscription ended.
+///
+/// The primary means of interacting with a [Subscription] is through the [futures::stream::Stream]
```

Comment:
does `futures::stream::Stream` show up as an "Important Trait Impl" in rustdoc the same way `Future` and `Iterator` do?

Reply:
I wasn't seeing it. there is just Trait Implementations.